src/main/scala/com/typesafe/conductr/client/ConductRController.scala

```diff
@@ -163,18 +169,24 @@ class ConductRController(uri: Uri, connectTimeout: Timeout)
   import ConductRController._
   import context.dispatcher
 
-  override def receive: Receive = {
-    case GetBundleInfoStream   => fetchBundleFlow(sender())
-    case request: LoadBundle   => loadBundle(request)
-    case request: StartBundle  => startBundle(request)
-    case request: StopBundle   => stopBundle(request)
-    case request: UnloadBundle => unloadBundle(request)
+  override def receive: Receive = service(uri)
+
+  def service(host: Uri): Receive = {
+    case GetBundleInfoStream   => fetchBundleFlow(sender(), host)
+    case request: LoadBundle   => loadBundle(request, host)
+    case request: StartBundle  => startBundle(request, host)
+    case request: StopBundle   => stopBundle(request, host)
+    case request: UnloadBundle => unloadBundle(request, host)
+    case SetControlServer(uri) =>
+      context.become(service(Uri(uri.toString)))
+      sender() ! "Success"
```

Comment:
"Success" is a bad message. I don't think we need to reply anyway, so please delete and move the `context.become` one line up after `=>`.